<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select API HOST for Target Data </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>541</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Request API</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Exporer File</string>
   </property>
   <widget class="QLineEdit" name="File_Path_Edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>input data file path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navi_file_path">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Do">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Do</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_path_print">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>380</y>
     <width>631</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Table View</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>631</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Graph View</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>531</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Graph View</string>
   </property>
   <widget class="QPushButton" name="DataShow">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
